content/php/concepts/math-functions/terms/is-finite/is-finite.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'is_finite()'
+Description: 'Evaluates whether a value is a legal finite number or not.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`is_finite()`** method evaluates whether the value is finite or not and returns a *boolean* value.
+
+
+## Syntax
+
+```pseudo
+$result = is_finite(value);
+```
+
+Returns `1` if `value` is finite, else returns `false/nothing`
+
+## Example
+
```

Comment:
This example doesn't produce any output or do anything with the result. Can we have a slightly more complex example, with a sentence introducing it and summarizing what it does?

Reply:
Okay.